tutorials/Hierarchical_modeling.ipynb

```diff
@@ -1446,14 +1445,14 @@
     "\n",
     "complete_pooling_queries = [complete_pooling_phi()]\n",
     "\n",
+    "# N.B., it is expected that this infer() call could be many minutes to an hour or more\n",
     "complete_pooling_samples = bm.GlobalNoUTurnSampler().infer(\n",
     "    queries=complete_pooling_queries,\n",
     "    observations=complete_pooling_observations,\n",
     "    num_samples=num_samples,\n",
     "    num_chains=4,\n",
     "    num_adaptive_samples=num_adaptive_samples,\n",
-    "    verbose=VerboseLevel.OFF,\n",
-    ")"
+    ")\n",
```

Comment:
Hi Brad, I think this trailing comma is why the internal test is failing: JSON doesn't allow the last item in a list to be followed by a comma. You can try removing this trailing comma and reimport to see if the test pass :)

Reply:
fixed in new commit